[2022-08-03 00:50:49,073] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ltv-pipeline.create_table manual__2022-08-03T00:50:45.322976+00:00 [queued]>
[2022-08-03 00:50:49,099] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ltv-pipeline.create_table manual__2022-08-03T00:50:45.322976+00:00 [queued]>
[2022-08-03 00:50:49,116] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 00:50:49,120] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-03 00:50:49,125] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 00:50:49,161] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): create_table> on 2022-08-03 00:50:45.322976+00:00
[2022-08-03 00:50:49,188] {standard_task_runner.py:52} INFO - Started process 95 to run task
[2022-08-03 00:50:49,197] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ltv-pipeline', 'create_table', 'manual__2022-08-03T00:50:45.322976+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/dag-test.py', '--cfg-path', '/tmp/tmp8uawy5i3', '--error-file', '/tmp/tmp87rhoh9w']
[2022-08-03 00:50:49,208] {standard_task_runner.py:80} INFO - Job 19: Subtask create_table
[2022-08-03 00:50:49,664] {task_command.py:371} INFO - Running <TaskInstance: ltv-pipeline.create_table manual__2022-08-03T00:50:45.322976+00:00 [running]> on host 4b5f29f024fc
[2022-08-03 00:50:49,831] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ltv-pipeline
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2022-08-03T00:50:45.322976+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-03T00:50:45.322976+00:00
[2022-08-03 00:50:49,869] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2022-08-03 00:50:49,885] {dbapi.py:231} INFO - Running statement: 
                CREATE TABLE IF NOT EXISTS users(
                    first_name TEXT NOT NULL,
                    last_name TEXT NOT NULL
                )
        , parameters: None
[2022-08-03 00:50:49,896] {postgres.py:94} INFO - NOTICE:  relation "users" already exists, skipping

[2022-08-03 00:50:49,944] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=ltv-pipeline, task_id=create_table, execution_date=20220803T005045, start_date=20220803T005049, end_date=20220803T005049
[2022-08-03 00:50:50,047] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-03 00:50:50,179] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
